### ISSUE TITLE
Improve direction for POST and PATCH

### ISSUE THREAD

**Issue by berziiii** [ga-wdi-boston/jquery-ajax-diagnostic#57]
Since we use `data = getFormFields(form)` in the ajax request during the lesson, we should specify to NOT use this during the diagnostic. Instead, clearer instructions should be added to create an object consisting of the data we want to send for these requests:

``` js
data = { 
  books: {
    title: "Title", 
    author: "Author", 
  }
};
```

**Comment by jrhorn424** [comment]
I asked @berziiii to file this. Thanks!

**Comment by RealWeeks** [comment]
@berziiii fixed in commit 091ff55

**Comment by RealWeeks** [comment]
Per new standards of closing with a commit. I am close this @berziiii
